lib/command/src/Obelisk/Command/Run.hs

```diff
@@ -89,21 +85,32 @@ profile profileBaseName = withProjectRoot "." $ \root -> do
   let -- Sane flags to enable by default, enable time profiling +
       -- closure heap profiling.
       rtsFlags = [ "+RTS", "-p", "-po" <> profileBaseName, "-hc", "-RTS" ]
-      nixBuildExpr = unlines $
-        [ "(with (import ./. {});"
-        , "let"
-        , "  exeSource = obelisk.nixpkgs.writeText \"ob-run\" ''"
-        , "module Main where"
-        , "import Control.Exception"
-        , "import Reflex.Profiled"
-        , "import System.Environment" ]
-        <> obRunImports <>
-        [ "main :: IO ()"
-        , "main = getArgs >>= \\args -> (Obelisk.Run.run (read (args !! 0)) (Obelisk.Run.runServeAsset (args !! 1)) Backend.backend Frontend.frontend) `finally` writeProfilingData \"" <> profileBaseName <> ".rprof\""
-        , "  '';"
-        , "in obelisk.nixpkgs.runCommand \"ob-run\" {"
-        , "  buildInputs = [ (profiled.ghc.ghcWithPackages (p: [ p.backend p.frontend])) ];"
-        , "} \"ghc -x hs -prof -fno-prof-auto ${exeSource} -o $out\")" ]
+      nixBuildExpr = [hereLit|
+with (import ./. {});
+let
+  exeSource = obelisk.nixpkgs.writeText "ob-run" ''
+module Main where
+
+import Control.Exception
+import Reflex.Profiled
+import System.Environment
+
+import qualified Obelisk.Run
+import qualified Frontend
+import qualified Backend
```

Comment:
You can still interpolate `obRunImports` if you use `iTrim` instead as the QQ.

Reply:
oh okay

My thought on this was that it didn't make sense to share these since the `ob run` and `ob profile` main expressions are now substantially different. Ideally these would be closer so that we don't accidentally break ob profile when ob run changes or vice versa.